include/igl/blue_noise.cpp

```diff
@@ -0,0 +1,390 @@
+// This file is part of libigl, a simple c++ geometry processing library.
+// 
+// Copyright (C) 2020 Rinat Abdrashitov <rinat@dgp.toronto.edu>
+// 
+// This Source Code Form is subject to the terms of the Mozilla Public License 
+// v. 2.0. If a copy of the MPL was not distributed with this file, You can 
+// obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "blue_noise.h"
+#include <algorithm>
+#include <cmath>
+#include <iostream>
+#include <cassert>
+#include <stdint.h>
+#include <vector>
+#include <Eigen/Core>
+
+//TODO: reimplement bridsons code below using Eigen
+namespace bridson
+{
+    using std::min;
```

Comment:
Do not alias std functions, prefer using `std::min` directly.

Reply:
fixed